api-specificatie/openapi.yaml

```diff
@@ -1343,22 +1379,35 @@ components:
         ligtAanOpenbareRuimte:
           $ref: '#/components/schemas/NEN3610ID'
           description: 'Een nummeraanduiding ligt aan een openbare ruimte.'
-        huisnummerMogelijkOnjuist:
-          type: boolean
-        huisletterMogelijkOnjuist:
-          type: boolean
-        huisnummertoevoegingMogelijkOnjuist:
-          type: boolean
-        postcodeMogelijkOnjuist:
-          type: boolean
-        typeAdresseerbaarObjectMogelijkOnjuist:
-          type: boolean
-        statusMogelijkOnjuist:
-          type: boolean
-        ligtInWoonplaatsMogelijkOnjuist:
-          type: boolean
-        ligtAanOpenbareRuimteMogelijkOnjuist:
-          type: boolean
+        mogelijkOnjuist:
+          $ref: '#/components/schemas/NummeraanduidingMogelijkOnjuist'
+    NummeraanduidingMogelijkOnjuist:
+        type: object
+        properties: 
+          huisnummerMogelijkOnjuist:
+            type: boolean
+            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+          huisletterMogelijkOnjuist:
+            type: boolean
+            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+          huisnummertoevoegingMogelijkOnjuist:
+            type: boolean
+            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+          postcodeMogelijkOnjuist:
+            type: boolean
+            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+          typeAdresseerbaarObjectMogelijkOnjuist:
```

Comment:
typeAdresseerbaarObject werd toch alleen type?

Reply:
Dit is de MogelijkOnjuist bij een Nummeraanduiding. DUs sowieso niet alleen type. 
Wij hebben ervoor gekozen om het typeAdresseerbaarObject daar niet in op te nemen (staat wel in de BAG). Willen we wel communiceren dat dit element eventueel mogelijkONjuist is. Dat wordt wel complex. Ik denk dat we typeAdresseerbaarObject niet in moglijkOnjuist op moeten nemen. @strijm wil jij dat ook met Pieter checken ?

Reply:
Wat Pieter betreft mag de MogelijkOnjuist property voor het typeAdresseerbaarObject eruit. Deze staat nl. ook niet in het mapping overzicht.